generators/generator-bot-calendar/generators/app/templates/dialogs/CreateEventDialog/CreateEventDialog.dialog

```diff
@@ -149,7 +150,8 @@
           "$designer": {
             "id": "DHvA61"
           },
-          "name": "CreateEventDialog.OnEvent_SetTitle"
+          "name": "CreateEventDialog.OnEvent_SetTitle",
+          "value": "=null"
```

Comment:
If this forces every skill developers to the add a "=null" to their bot, then it would be tragic. :(

Reply:
Note to self - bring this up with the SDK team. Not as a P0 but probably as a P1 issue to fix.